src/aead/less_safe_key.rs

```diff
@@ -44,6 +44,26 @@ impl LessSafeKey {
         })
     }
 
+    /// Like [open_in_place](Self::open_in_place), except it accepts an arbitrary authentication tag.
+    #[inline]
+    pub fn open_in_place_seperate_tag<'in_out, A>(
+        &self,
+        nonce: Nonce,
+        aad: Aad<A>,
+        tag: Tag,
+        in_out: &'in_out mut [u8],
```

Comment:
Why not keep the flexibility of shifting the output like the other `open_in_place` functions why adding a `src: RangeFrom<usize>` parameter? I don't see a reason to drop that functionality with the new API.

Reply:
This was mostly an oversight on my part implementing it. I've added it back.